src/main/java/seedu/intrack/model/internship/UniqueInternshipList.java

```diff
@@ -99,6 +101,8 @@ public void setInternships(List<Internship> internships) {
         internalList.setAll(internships);
     }
 
+
+
```

Comment:
Unnecessary white space.

Reply:
Thanks for pointing that out, will be fixed.